docs/generator/links_to_types.go

```diff
@@ -101,7 +101,7 @@ func acceptingComponentsSection(componentName string, meta metadata.Metadata) st
 	for _, acceptedDataType := range meta.AllTypesAccepted() {
 		if list := allComponentsThatExport(acceptedDataType); len(list) > 0 {
 			section += fmt.Sprintf(
-				"- Components that export [%s]({{< relref \"../compatibility/%s\" >}})\n",
+				"- Components that export [%s](../compatibility%s)\n",
```

Comment:
We need to go up one more directory to get to the parent of `compatibility` from these component pages.

I think we've lost the trailing slash accidentally. Although it was not desirable in the `relref` (https://grafana.com/docs/writers-toolkit/review/doc-validator/errata/#relref-has-trailing-slash), it avoids a redirect if you are using URLs instead.

```suggestion
				"- Components that export [%s](../../compatibility/%s)\n",
```